huaweicloud/services/acceptance/dws/data_source_huaweicloud_dws_workload_queues_test.go

```diff
@@ -0,0 +1,66 @@
+package dws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccWorkloadQueuesDataSource_basic(t *testing.T) {
+	resourceName := "data.huaweicloud_dws_workload_queues.test"
+	dc := acceptance.InitDataSourceCheck(resourceName)
+	name := acceptance.RandomAccResourceName()
+	// The cluster password requires a minimum length of 12 characters, and the string 'gap' is used to fill in the gap.
+	password := acceptance.RandomPassword() + "gap"
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccWorkloadQueuesDataSourceBasic(name, password),
+				Check: resource.ComposeTestCheckFunc(
+					dc.CheckResourceExists(),
+					resource.TestCheckResourceAttrSet(resourceName, "queues.#"),
+					resource.TestCheckOutput("name_filter_is_useful", "true"),
+					resource.TestCheckOutput("no_filter_is_useful", "true"),
+				),
+			},
+		},
+	})
+}
+
+func testAccWorkloadQueuesDataSourceBasic(name, password string) string {
```

Comment:
did we have any test to check what is happened if the query input parameter is invalid?

Reply:
done